union-type.js

```diff
@@ -1,78 +1,130 @@
 var curryN = require('ramda/src/curryN');
 
-function isString(s) { return typeof s === 'string'; }
-function isNumber(n) { return typeof n === 'number'; }
-function isBoolean(b) { return typeof b === 'boolean'; }
-function isObject(value) {
-  var type = typeof value;
-  return !!value && (type == 'object' || type == 'function');
-}
-function isFunction(f) { return typeof f === 'function'; }
-var isArray = Array.isArray || function(a) { return 'length' in a; };
+if (process.env.NODE_ENV !== 'production') {
+  var isString = function(s) { return typeof s === 'string'; };
+  var isNumber = function(n) { return typeof n === 'number'; };
+  var isBoolean = function(b) { return typeof b === 'boolean'; };
+  var isObject = function(value) {
+    var type = typeof value;
+    return !!value && (type == 'object' || type == 'function');
+  };
+  var isFunction = function(f) { return typeof f === 'function'; };
```

Comment:
Oh I am sorry you are of course right. I somehow thought I had seen babel invocation in package.json but that wasn't babel.. ;)